src/java.base/share/classes/java/io/BufferedInputStream.java

```diff
@@ -50,6 +50,12 @@
  * reread before new bytes are  taken from
  * the contained input stream.
  *
+ * <p> More than one instance of {@code BufferedInputStream} should not be
+ * used with the same underlying {@code InputStream} instance. Doing
+ * so can cause the {@code BufferedInputStream} instances to return an
+ * incorrect result since each instance of {@code BufferedInputStream}
+ * maintains its own state.
+ *
```

Comment:
It might be clearer to just say that once wrapped with a BIS, the underlying input stream should not be used directly or wrapped with another stream. This is potentially something for an apiNote as it's more about API usage,

Reply:
Verbiage changed and converted to an `apiNote` in 503d123.